hubblestack/audit/misc.py

```diff
@@ -1112,6 +1121,33 @@ def _check_sshd_parameters(block_id, block_dict, extra_args=None):
     else:
         return "The comparetype: " + comparetype + " not found. It can be 'regex' or 'only'. Please check."
 
+def _test_mount_attrs(block_id, block_dict, extra_args=None):
+    """
+    Ensure that a given directory is mounted with appropriate attributes
+    If check_type is soft, then in absence of volume, True will be returned
+    If check_type is hard, then in absence of volume, False will be returned
+    """
+    mount_name = runner_utils.get_param_for_module(block_id, block_dict, 'mount_name')
+    attribute = runner_utils.get_param_for_module(block_id, block_dict, 'attribute')
+    check_type = runner_utils.get_param_for_module(block_id, block_dict, 'check_type', 'hard')
+
+    # check that the path exists on system
+    command = 'test -e ' + mount_name
```

Comment:
please use join for string concatenation.

Reply:
I don't think join is necessary here. That just introduces another datastructure.